extensions/sql-migration/src/api/azure.ts

```diff
@@ -640,8 +816,100 @@ export interface StartDatabaseMigrationRequest {
 
 export interface StartDatabaseMigrationResponse {
 	status: number,
-	databaseMigration: DatabaseMigration
-	asyncUrl: string
+	databaseMigration: DatabaseMigration,
+	asyncUrl: string,
+}
+
+export enum Kind {
```

Comment:
nit: I guess this name `Kind` matches the parameter name `kind` in the request body, but could we rename this enum to be a bit more descriptive? e.g. `TargetKind`, or something like that

Reply:
also I might be mistaken (I probably am) but I could've sworn a similar enum was defined elsewhere in the past? Just want to double check there's no duplicate definitions

Reply:
There are 3 that seem to be similar, Kind/AzureResourceKind, MigrationTargetType and SQLTargetAssetType but all are different values...